core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -629,6 +629,10 @@ enum class UniqueType(
     PillageYieldFixed("Pillaging this improvement yields [stats]", UniqueTarget.Improvement),
     Irremovable("Irremovable", UniqueTarget.Improvement),
     AutomatedUnitsWillNotReplace("Will not be replaced by automated units", UniqueTarget.Improvement),
+    ImprovesResources("Improves [resourceFilter] resource in this tile", UniqueTarget.Improvement, flags = UniqueFlag.setOfNoConditionals,
```

Comment:
I think "provides" instead of "improves" is clearer as to what this actually does

Reply:
👍